#tactile sensor data in 2D grid
Header header
sdh2_hand/TactileMatrix[] tactile_matrix
